docker-storage-setup.sh

```diff
@@ -1169,6 +1169,11 @@ if [ -e /etc/sysconfig/docker-storage-setup ]; then
   source /etc/sysconfig/docker-storage-setup
 fi
 
+if [ ! -z "$EXTRA_DOCKER_STORAGE_OPTIONS" ]; then
+    Info "EXTRA_DOCKER_STORAGE_OPTIONS is deprecated, please use EXTRA_STORAGE_OPTIONS"
+    EXTRA_STORAGE_OPTIONS=${EXTRA_DOCKER_STORAGE_OPTIONS}
+fi
```

Comment:
We also need to have a check to make sure user can't specify both old and new options. Otherwise with current code, old option will overwrite new option.

We already have a function called check_storage_options() and I think these checks can go in there?